### PR TITLE
Add more tests for password authentication.

### PR DESCRIPTION
**Pull request by brendandburns**

### CODE REVIEW

**Comment by drubin** [comment]
@brendandburns I think the dependancies all belong as devDependancies.

**Comment by brendandburns** [comment]
@drubin thanks for the quick turn around. Fixed (and fixed lint too)

File: src/config_test.ts [hunk 1]
```diff
@@ -30,20 +30,24 @@ function validateFileLoad(kc: KubeConfig) {
...
     expect(user2.name).to.equal('user2');
     expect(user2.certData).to.equal('VVNFUjJfQ0FEQVRB');
     expect(user2.keyData).to.equal('VVNFUjJfQ0tEQVRB');
-
+    expect(user3.name).to.equal('user3');
+    expect(user3.username).to.equal('foo');
+    expect(user3.password).to.equal('bar');
     // check contexts
-    expect(kc.contexts.length).to.equal(2, 'there are two contexts');
+    expect(kc.contexts.length).to.equal(3, 'there are two contexts');
```
Comment (drubin) [inline src/config_test.ts]: Silly nitpick but this should now be 3.

Comment (brendandburns) [reply src/config_test.ts]: fixed.

**Comment by brendandburns** [comment]
Comment addressed (and I cleaned up an import to new style too :)

**Pull request merged**